src/scene/lightmapper.js

```diff
@@ -244,641 +491,526 @@ class Lightmapper {
      * Only lights with bakeDir=true will be used for generating the dominant light direction. Defaults to
      * pc.BAKE_COLORDIR.
      */
-    bake(nodes, mode) {
+    bake(nodes, mode = BAKE_COLORDIR) {
+
+        let device = this.device;
+        const startTime = now();
```

Comment:
Why was this moved outside the `// #ifdef PROFILER` block?

Reply:
we have a request to make performance stats available in all stats .. and considering this costs us nothing, it's a good start.